ipu-plugin/pkg/ipuplugin/deviceplugin.go

```diff
@@ -35,6 +35,9 @@ var (
 	maxVfsSupported  = 64
 )
 
+var hostVfDevs map[string]*pb.Device
```

Comment:
can we please add comment what these maps contains? From the code it seems like one is file to device while the other one is pciaddr ? Do the key values look similar at the end or are they different? Maybe add examples to be clear?